src/main/java/org/spongepowered/common/mixin/realtime/entity/EntityAgeableMixin_RealTime.java

```diff
@@ -45,7 +45,10 @@
         }
         // Subtract the one the original update method added
         final int diff = (int) ((RealTimeTrackingBridge) this.world).realTimeBridge$getRealTimeTicks() - 1;
-        this.setGrowingAge(Math.min(0, age + diff));
+        // age is positive ? subtract : add
+        final int new_age = age - (int) Math.signum(age) * diff;
```

Comment:
Rather than do this which is confusing to read, I'd much rather make this a simple if statement - something like:

```java
if (age > 0) {
    this.setGrowingAge(Math.max(0, age - diff));
} else {
    this.setGrowingAge(Math.min(0, age + diff));
}
```

That way it's a lot clearer about what's happening. Could also have a branch that bypasses these things completely if `diff = 0`

Reply:
Agreed. Didn't think of that for some reason :D